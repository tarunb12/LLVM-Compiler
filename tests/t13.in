int func() {
    return "invalid return type"
}

int main() {
    printf(func())
}