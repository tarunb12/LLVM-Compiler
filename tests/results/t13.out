Error: File "tests/t13.in": Invalid return type specified in the function func: "string", expected "int" in "return "invalid return type"".